src/calendar/view/CalendarTimeIndicator.ts

```diff
@@ -0,0 +1,49 @@
+import m, { Children, Component, Vnode } from "mithril"
+import { px, size } from "../../gui/size.js"
+import { theme } from "../../gui/theme.js"
+
+export type CalendarTimeIndicatorAttrs = {
+	/**
+	 * offset in pixels from the top to draw the indicator
+	 */
+	offset: number
+
+	/**
+	 * display a circle in addition to a line
+	 */
+	circle: boolean
+}
+
+/**
+ * Indicator used for indicating the current time relative to the current date in the calendar view.
+ */
+export class CalendarTimeIndicator implements Component<CalendarTimeIndicatorAttrs> {
+	view({ attrs }: Vnode<CalendarTimeIndicatorAttrs>): Children {
+		return [
+			m(".abs", {
+				"aria-hidden": "true",
+				style: {
+					top: px(attrs.offset),
+					left: 0,
+					right: 0,
+					height: px(size.calendar_day_event_padding),
+					background: theme.content_accent,
+				},
+			}),
+			attrs.circle
+				? m(`.abs.circle`, {
+						"aria-hidden": "true",
+						style: {
+							top: px(attrs.offset),
+							left: 0,
+							height: px(size.icon_size_small),
+							width: px(size.icon_size_small),
+							background: theme.content_accent,
+							"margin-top": "-5px",
+							"margin-left": "-7px",
```

Comment:
There is a preference for using the `px()` function to generate the strings.
```suggestion
							"margin-top": px(-5),
							"margin-left": px(-7),
```